addons/pos_customer_display/controllers/main.py

```diff
@@ -0,0 +1,20 @@
+import werkzeug
+
+from odoo import http
+from odoo.http import request
+
+class PosCustomerDisplay(http.Controller):
+    @http.route("/pos-customer-display/<access_token>", auth="public", type="http", website=True)
```

Comment:
Can you conform to snake case

Reply:
But this is what we do all over the place 
![image](https://github.com/odoo/odoo/assets/118446179/a38189d8-0ec3-447a-bd0f-129172c584fb)
![image](https://github.com/odoo/odoo/assets/118446179/21cda92a-4a3b-48b0-ad5c-4617864f7012)

Reply:
grep on the code for controllers and compare
maybe SEO is friendly for human

Reply:
I'll clarify some things regarding "routes":

1 - You simply have to follow the Odoo guidelines for creating routes [1]. So snake case. As applied by all other Odoo modules, perhaps not some POS modules.

2 - To justify your use of hyphen you show me a URL of an e-commerce product which is made to be SEO friendly. Which has nothing to do with constructing a "route" in python. [2]

![screenshot-59223816-master-all runbot106 odoo com-2024 03 04-10_34_05](https://github.com/odoo/odoo/assets/38907656/8d6308a1-ae63-435c-8631-f843170edecd)

[1] https://www.odoo.com/documentation/17.0/developer/reference/backend/http.html

[2] https://www.youtube.com/live/thGLpqV5wtE?si=OrQhnb5mdOmLCdiw&t=895